src/ServiceContainer/SymfonyExtension.php

```diff
@@ -90,13 +91,12 @@ public function configure(ArrayNodeDefinition $builder): void
         $builder
             ->addDefaultsIfNotSet()
                 ->children()
+                    ->scalarNode('env_file')->defaultValue('.env')->end()
```

Comment:
If it was null by default and the rest of config keeps working the same way it was, then it's a win.
If someone defines `env_file` we can use its variables first with a fallback to these `env` and `debug` configuration values.

Reply:
ok! I will update that !